2023-10-01 00:35:52,855 - INFO - Program started successfully!
2023-10-01 00:38:13,413 - INFO - Program started successfully!
2023-10-01 00:38:13,604 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 00:38:16,994 - ERROR - Error while calling Moodle API: Invalid URL 'localhost/webservice/rest/server.php': No scheme supplied. Perhaps you meant https://localhost/webservice/rest/server.php?
2023-10-01 00:38:16,994 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): cannot access local variable 'response' where it is not associated with a value
2023-10-01 00:39:00,076 - ERROR - Error while calling Moodle API: Invalid URL 'localhost/webservice/rest/server.php': No scheme supplied. Perhaps you meant https://localhost/webservice/rest/server.php?
2023-10-01 00:39:00,076 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): cannot access local variable 'response' where it is not associated with a value
2023-10-01 00:39:15,020 - INFO - Program started successfully!
2023-10-01 00:39:15,206 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 00:39:26,507 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected'})
2023-10-01 00:39:30,593 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected'})
2023-10-01 00:39:34,697 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected'})
2023-10-01 00:39:38,782 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected'})
2023-10-01 00:39:42,843 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected'})
2023-10-01 00:39:46,916 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected'})
2023-10-01 00:39:50,971 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected'})
2023-10-01 00:41:43,895 - INFO - Program started successfully!
2023-10-01 00:41:44,092 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 00:41:55,413 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected'})
2023-10-01 00:41:59,518 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected'})
2023-10-01 00:42:03,606 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected'})
2023-10-01 00:42:07,686 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected'})
2023-10-01 00:42:47,353 - INFO - Program started successfully!
2023-10-01 00:42:47,553 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 00:42:59,122 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: боровлев(cleaned value: ))", 'debuginfo': "The field 'username' value is invalid: боровлев(cleaned value: )"})
2023-10-01 00:43:03,222 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: боровлев1(cleaned value: 1))", 'debuginfo': "The field 'username' value is invalid: боровлев1(cleaned value: 1)"})
2023-10-01 00:43:07,364 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: боровлев12(cleaned value: 12))", 'debuginfo': "The field 'username' value is invalid: боровлев12(cleaned value: 12)"})
2023-10-01 00:44:21,107 - INFO - Program started successfully!
2023-10-01 00:44:21,291 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 00:44:36,736 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'moodle_exception', 'errorcode': '<div>Passwords must have at least 1 upper case letter(s).</div>', 'message': 'error/<div>Passwords must have at least 1 upper case letter(s).</div>'})
2023-10-01 00:44:40,871 - ERROR - Error while creating user with email piratisrael@gmail.com: list index out of range
2023-10-01 00:47:23,113 - INFO - Program started successfully!
2023-10-01 00:47:23,317 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 00:47:25,995 - ERROR - Error while getting data from the sheet Осокин: the given 'expected_headers' contains unknown headers: {'Трек-номер'}
2023-10-01 00:48:42,719 - INFO - Program started successfully!
2023-10-01 00:48:42,904 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 00:48:58,524 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'moodle_exception', 'errorcode': '<div>The password must have at least 1 special character(s) such as *, -, or #.</div>', 'message': 'error/<div>The password must have at least 1 special character(s) such as *, -, or #.</div>'})
2023-10-01 00:53:18,412 - INFO - Program started successfully!
2023-10-01 00:53:18,620 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 00:53:46,948 - INFO - New course created: Modern aspekts of aspekts
2023-10-01 00:53:55,881 - INFO - New user registered: Borolev Dmitriy Aleksandrovich with login borolev and enrolled to the course Modern aspekts of aspekts
2023-10-01 00:54:19,670 - ERROR - Error while getting data from the sheet Осокин: the given 'expected_headers' contains unknown headers: {'Трек-номер'}
2023-10-01 00:54:42,405 - ERROR - Error while getting data from the sheet Осокин: the given 'expected_headers' contains unknown headers: {'Трек-номер'}
2023-10-01 00:59:36,090 - INFO - Program started successfully!
2023-10-01 00:59:36,281 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 00:59:52,381 - INFO - New user registered: Borolev Dmitriy Aleksandrovich with login borolev and enrolled to the course Modern aspekts of aspekts
2023-10-01 00:59:52,381 - ERROR - Error while sending mail: [Errno 2] No such file or directory: 'letters/enrollment.html.html'. Trying again
2023-10-01 00:59:52,382 - INFO - Restarting program...
2023-10-01 01:01:18,696 - INFO - Program started successfully!
2023-10-01 01:01:18,902 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 01:01:34,170 - INFO - New user registered: Borolev Dmitriy Aleksandrovich with login api and enrolled to the course Modern aspekts of aspekts
2023-10-01 01:03:38,896 - INFO - Program started successfully!
2023-10-01 01:03:39,091 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 01:04:10,972 - INFO - New user registered: Borolev Dmitriy Aleksandrovich with login borolev1 and enrolled to the course Modern aspekts of aspekts
2023-10-01 01:10:40,522 - INFO - Program started successfully!
2023-10-01 01:10:40,718 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 01:10:56,213 - INFO - New course created: Old aspekt of pivo
2023-10-01 01:11:05,096 - INFO - New user registered: Borolev Dmitriy Aleksandrovich with login borolev1 and enrolled to the course Old aspekt of pivo
2023-10-01 10:49:19,002 - INFO - Program started successfully!
2023-10-01 10:49:19,180 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 10:49:23,270 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'})
2023-10-01 10:49:23,270 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0
2023-10-01 10:49:47,072 - INFO - Program started successfully!
2023-10-01 10:49:47,251 - INFO - file_cache is only supported with oauth2client<4.0.0
2023-10-01 10:49:50,486 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'})
2023-10-01 10:49:50,486 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0
2023-10-01 10:51:18,144 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:509
2023-10-01 10:51:18,336 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 10:51:21,522 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:327
2023-10-01 10:51:21,522 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:504
2023-10-01 10:56:41,755 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:509
2023-10-01 10:56:41,961 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 10:56:45,197 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:327
2023-10-01 10:56:45,198 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:504
2023-10-01 11:00:00,456 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:510
2023-10-01 11:00:00,656 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:00:04,494 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 11:00:04,495 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:505
2023-10-01 11:02:23,806 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:509
2023-10-01 11:02:24,014 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:02:27,100 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:327
2023-10-01 11:02:27,101 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:504
2023-10-01 11:03:30,284 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:509
2023-10-01 11:03:30,505 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:03:33,851 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:504
2023-10-01 11:05:28,909 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:509
2023-10-01 11:05:29,131 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:05:32,438 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 11:05:32,438 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:504
2023-10-01 11:08:15,343 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:510
2023-10-01 11:08:15,546 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:08:18,776 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:327
2023-10-01 11:08:18,776 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:505
2023-10-01 11:12:14,368 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:510
2023-10-01 11:12:14,573 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:12:18,076 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:327
2023-10-01 11:12:18,076 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:505
2023-10-01 11:26:04,720 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:514
2023-10-01 11:26:04,932 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:26:10,879 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (users => Invalid parameter value detected (Only arrays accepted. The bad value is: 'username'): Only arrays accepted. The bad value is: 'username')", 'debuginfo': "users => Invalid parameter value detected (Only arrays accepted. The bad value is: 'username'): Only arrays accepted. The bad value is: 'username'"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:331
2023-10-01 11:26:10,927 - ERROR - Error while creating user with email info@info.info: list index out of range - C:\Users\epira\Downloads\MoodleAPI-master\main.py:421
2023-10-01 11:26:11,195 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): {'code': 400, 'message': 'Invalid values[1][0]: list_value {\n  values {\n    string_value: "error"\n  }\n  values {\n    string_value: ""\n  }\n  values {\n    string_value: "ERROR - there is problem while user registration"\n  }\n}\n', 'status': 'INVALID_ARGUMENT'} - C:\Users\epira\Downloads\MoodleAPI-master\main.py:509
2023-10-01 11:33:37,395 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:515
2023-10-01 11:33:37,600 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:33:43,022 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:332
2023-10-01 11:33:43,023 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:510
2023-10-01 11:37:13,094 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:508
2023-10-01 11:37:13,304 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:37:16,574 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 11:37:16,575 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:503
2023-10-01 11:37:36,260 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:509
2023-10-01 11:37:36,445 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:37:39,498 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:327
2023-10-01 11:37:39,498 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:504
2023-10-01 11:46:58,348 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:513
2023-10-01 11:46:58,550 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:47:01,763 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:331
2023-10-01 11:47:01,763 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:508
2023-10-01 11:47:53,313 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:524
2023-10-01 11:47:53,485 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:47:56,623 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:519
2023-10-01 11:48:39,808 - ERROR - Incorrect email - test@test.test: The part after the @-sign is a special-use or reserved name that cannot be used with email. - C:\Users\epira\Downloads\MoodleAPI-master\main.py:401
2023-10-01 11:48:40,030 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): {'code': 400, 'message': 'Invalid values[1][0]: list_value {\n  values {\n    string_value: "error"\n  }\n  values {\n    string_value: ""\n  }\n  values {\n    string_value: "ERROR - incorrect email!"\n  }\n}\n', 'status': 'INVALID_ARGUMENT'} - C:\Users\epira\Downloads\MoodleAPI-master\main.py:519
2023-10-01 11:49:23,924 - ERROR - Incorrect email - test@test.test: The part after the @-sign is a special-use or reserved name that cannot be used with email. - C:\Users\epira\Downloads\MoodleAPI-master\main.py:401
2023-10-01 11:49:24,155 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): {'code': 400, 'message': 'Invalid values[1][0]: list_value {\n  values {\n    string_value: "error"\n  }\n  values {\n    string_value: ""\n  }\n  values {\n    string_value: "ERROR - incorrect email!"\n  }\n}\n', 'status': 'INVALID_ARGUMENT'} - C:\Users\epira\Downloads\MoodleAPI-master\main.py:519
2023-10-01 11:50:26,749 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:524
2023-10-01 11:50:26,935 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:50:30,438 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:519
2023-10-01 11:51:14,901 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:509
2023-10-01 11:51:15,096 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:51:18,305 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': 'Invalid parameter value detected (Unexpected keys (values) detected in parameter array.)', 'debuginfo': 'Unexpected keys (values) detected in parameter array.'}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:327
2023-10-01 11:51:18,306 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:504
2023-10-01 11:51:47,192 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:509
2023-10-01 11:51:47,357 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:51:50,696 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:504
2023-10-01 11:52:23,542 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:509
2023-10-01 11:52:23,746 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:52:26,838 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:504
2023-10-01 11:52:54,405 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:511
2023-10-01 11:52:54,597 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:52:57,641 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:506
2023-10-01 11:53:44,782 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:510
2023-10-01 11:53:45,284 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:53:48,549 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:505
2023-10-01 11:54:18,664 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:510
2023-10-01 11:54:18,849 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:54:21,894 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:505
2023-10-01 11:54:31,868 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:510
2023-10-01 11:54:32,086 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:54:35,070 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:505
2023-10-01 11:56:34,365 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:510
2023-10-01 11:56:34,542 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:56:37,569 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:505
2023-10-01 11:56:48,621 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:510
2023-10-01 11:56:48,781 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:56:51,884 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:505
2023-10-01 11:57:09,607 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:510
2023-10-01 11:57:09,815 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:57:13,122 - INFO - New course created: Test12312 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:437
2023-10-01 11:57:13,205 - ERROR - Error while creating course Test12312: 0 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:444
2023-10-01 11:57:13,435 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): {'code': 400, 'message': 'Invalid values[1][0]: list_value {\n  values {\n    string_value: "error"\n  }\n  values {\n    string_value: ""\n  }\n  values {\n    string_value: "ERROR - there is problem while creating course"\n  }\n}\n', 'status': 'INVALID_ARGUMENT'} - C:\Users\epira\Downloads\MoodleAPI-master\main.py:505
2023-10-01 11:57:57,870 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:509
2023-10-01 11:57:58,054 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 11:58:01,164 - INFO - New course created: Test123121 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:436
2023-10-01 11:58:02,109 - INFO - New user registered: Borolev Dmitriy Aleksandrovich with login borolev123 and enrolled to the course Test123121 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:453
2023-10-01 12:04:11,945 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:508
2023-10-01 12:04:12,123 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 12:04:15,668 - INFO - New course created: Test123122 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:435
2023-10-01 12:04:16,412 - INFO - New user registered: Alexandrov Andrey with login alexandrov and enrolled to the course Test123122 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 12:04:17,640 - INFO - New user registered: Alexandrov Andrey with login alexandrov1 and enrolled to the course Test123122 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 12:04:18,799 - INFO - New course created: Test123123 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:435
2023-10-01 12:04:19,349 - INFO - New user registered: Alexandrov Andrey with login alexandrov2 and enrolled to the course Test123123 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 12:04:20,679 - INFO - New course created: Test123124 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:435
2023-10-01 12:04:21,246 - INFO - New user registered: Alexandrov Andrey with login alexandrov3 and enrolled to the course Test123124 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 12:04:22,620 - INFO - New course created: Test123125 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:435
2023-10-01 12:04:23,044 - INFO - New user registered: Alexandrov Andrey with login alexandrov4 and enrolled to the course Test123125 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 12:04:24,486 - INFO - New course created: Test123126 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:435
2023-10-01 12:04:25,027 - INFO - New user registered: Alexandrov Andrey with login alexandrov5 and enrolled to the course Test123126 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 12:07:11,689 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:508
2023-10-01 12:07:11,862 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 12:07:15,666 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov(cleaned value: a_alexandrov))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov(cleaned value: a_alexandrov)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:15,714 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov1(cleaned value: a_alexandrov1))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov1(cleaned value: a_alexandrov1)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:15,760 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov2(cleaned value: a_alexandrov2))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov2(cleaned value: a_alexandrov2)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:15,810 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov3(cleaned value: a_alexandrov3))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov3(cleaned value: a_alexandrov3)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:15,857 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov4(cleaned value: a_alexandrov4))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov4(cleaned value: a_alexandrov4)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:15,903 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov5(cleaned value: a_alexandrov5))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov5(cleaned value: a_alexandrov5)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:15,936 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov6(cleaned value: a_alexandrov6))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov6(cleaned value: a_alexandrov6)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:15,984 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov7(cleaned value: a_alexandrov7))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov7(cleaned value: a_alexandrov7)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,016 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov8(cleaned value: a_alexandrov8))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov8(cleaned value: a_alexandrov8)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,061 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov9(cleaned value: a_alexandrov9))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov9(cleaned value: a_alexandrov9)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,107 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov10(cleaned value: a_alexandrov10))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov10(cleaned value: a_alexandrov10)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,155 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov11(cleaned value: a_alexandrov11))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov11(cleaned value: a_alexandrov11)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,203 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov12(cleaned value: a_alexandrov12))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov12(cleaned value: a_alexandrov12)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,253 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov13(cleaned value: a_alexandrov13))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov13(cleaned value: a_alexandrov13)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,299 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov14(cleaned value: a_alexandrov14))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov14(cleaned value: a_alexandrov14)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,347 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov15(cleaned value: a_alexandrov15))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov15(cleaned value: a_alexandrov15)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,395 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov16(cleaned value: a_alexandrov16))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov16(cleaned value: a_alexandrov16)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,445 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov17(cleaned value: a_alexandrov17))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov17(cleaned value: a_alexandrov17)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,489 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov18(cleaned value: a_alexandrov18))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov18(cleaned value: a_alexandrov18)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,538 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov19(cleaned value: a_alexandrov19))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov19(cleaned value: a_alexandrov19)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,585 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov20(cleaned value: a_alexandrov20))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov20(cleaned value: a_alexandrov20)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,633 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov21(cleaned value: a_alexandrov21))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov21(cleaned value: a_alexandrov21)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,681 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov22(cleaned value: a_alexandrov22))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov22(cleaned value: a_alexandrov22)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,729 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov23(cleaned value: a_alexandrov23))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov23(cleaned value: a_alexandrov23)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,777 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov24(cleaned value: a_alexandrov24))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov24(cleaned value: a_alexandrov24)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,825 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov25(cleaned value: a_alexandrov25))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov25(cleaned value: a_alexandrov25)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,871 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov26(cleaned value: a_alexandrov26))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov26(cleaned value: a_alexandrov26)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,903 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov27(cleaned value: a_alexandrov27))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov27(cleaned value: a_alexandrov27)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,952 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov28(cleaned value: a_alexandrov28))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov28(cleaned value: a_alexandrov28)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:16,997 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov29(cleaned value: a_alexandrov29))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov29(cleaned value: a_alexandrov29)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,046 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov30(cleaned value: a_alexandrov30))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov30(cleaned value: a_alexandrov30)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,093 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov31(cleaned value: a_alexandrov31))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov31(cleaned value: a_alexandrov31)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,140 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov32(cleaned value: a_alexandrov32))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov32(cleaned value: a_alexandrov32)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,186 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov33(cleaned value: a_alexandrov33))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov33(cleaned value: a_alexandrov33)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,234 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov34(cleaned value: a_alexandrov34))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov34(cleaned value: a_alexandrov34)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,283 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov35(cleaned value: a_alexandrov35))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov35(cleaned value: a_alexandrov35)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,329 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov36(cleaned value: a_alexandrov36))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov36(cleaned value: a_alexandrov36)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,376 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov37(cleaned value: a_alexandrov37))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov37(cleaned value: a_alexandrov37)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,423 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov38(cleaned value: a_alexandrov38))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov38(cleaned value: a_alexandrov38)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,470 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov39(cleaned value: a_alexandrov39))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov39(cleaned value: a_alexandrov39)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,503 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov40(cleaned value: a_alexandrov40))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov40(cleaned value: a_alexandrov40)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,549 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov41(cleaned value: a_alexandrov41))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov41(cleaned value: a_alexandrov41)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,597 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov42(cleaned value: a_alexandrov42))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov42(cleaned value: a_alexandrov42)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,645 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov43(cleaned value: a_alexandrov43))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov43(cleaned value: a_alexandrov43)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,677 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov44(cleaned value: a_alexandrov44))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov44(cleaned value: a_alexandrov44)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,709 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov45(cleaned value: a_alexandrov45))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov45(cleaned value: a_alexandrov45)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,756 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov46(cleaned value: a_alexandrov46))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov46(cleaned value: a_alexandrov46)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,804 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov47(cleaned value: a_alexandrov47))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov47(cleaned value: a_alexandrov47)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,850 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov48(cleaned value: a_alexandrov48))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov48(cleaned value: a_alexandrov48)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,896 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov49(cleaned value: a_alexandrov49))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov49(cleaned value: a_alexandrov49)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,944 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov50(cleaned value: a_alexandrov50))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov50(cleaned value: a_alexandrov50)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:17,991 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov51(cleaned value: a_alexandrov51))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov51(cleaned value: a_alexandrov51)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,039 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov52(cleaned value: a_alexandrov52))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov52(cleaned value: a_alexandrov52)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,086 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov53(cleaned value: a_alexandrov53))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov53(cleaned value: a_alexandrov53)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,133 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov54(cleaned value: a_alexandrov54))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov54(cleaned value: a_alexandrov54)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,182 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov55(cleaned value: a_alexandrov55))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov55(cleaned value: a_alexandrov55)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,230 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov56(cleaned value: a_alexandrov56))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov56(cleaned value: a_alexandrov56)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,279 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov57(cleaned value: a_alexandrov57))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov57(cleaned value: a_alexandrov57)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,326 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov58(cleaned value: a_alexandrov58))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov58(cleaned value: a_alexandrov58)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,373 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov59(cleaned value: a_alexandrov59))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov59(cleaned value: a_alexandrov59)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,405 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov60(cleaned value: a_alexandrov60))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov60(cleaned value: a_alexandrov60)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,451 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov61(cleaned value: a_alexandrov61))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov61(cleaned value: a_alexandrov61)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,499 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov62(cleaned value: a_alexandrov62))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov62(cleaned value: a_alexandrov62)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,546 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov63(cleaned value: a_alexandrov63))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov63(cleaned value: a_alexandrov63)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,596 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov64(cleaned value: a_alexandrov64))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov64(cleaned value: a_alexandrov64)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,641 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov65(cleaned value: a_alexandrov65))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov65(cleaned value: a_alexandrov65)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,690 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov66(cleaned value: a_alexandrov66))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov66(cleaned value: a_alexandrov66)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,737 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov67(cleaned value: a_alexandrov67))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov67(cleaned value: a_alexandrov67)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,785 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov68(cleaned value: a_alexandrov68))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov68(cleaned value: a_alexandrov68)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,832 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov69(cleaned value: a_alexandrov69))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov69(cleaned value: a_alexandrov69)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,882 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov70(cleaned value: a_alexandrov70))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov70(cleaned value: a_alexandrov70)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,928 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov71(cleaned value: a_alexandrov71))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov71(cleaned value: a_alexandrov71)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:18,976 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov72(cleaned value: a_alexandrov72))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov72(cleaned value: a_alexandrov72)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,024 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov73(cleaned value: a_alexandrov73))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov73(cleaned value: a_alexandrov73)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,073 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov74(cleaned value: a_alexandrov74))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov74(cleaned value: a_alexandrov74)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,106 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov75(cleaned value: a_alexandrov75))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov75(cleaned value: a_alexandrov75)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,150 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov76(cleaned value: a_alexandrov76))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov76(cleaned value: a_alexandrov76)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,197 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov77(cleaned value: a_alexandrov77))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov77(cleaned value: a_alexandrov77)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,245 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov78(cleaned value: a_alexandrov78))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov78(cleaned value: a_alexandrov78)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,292 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov79(cleaned value: a_alexandrov79))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov79(cleaned value: a_alexandrov79)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,340 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov80(cleaned value: a_alexandrov80))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov80(cleaned value: a_alexandrov80)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,371 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov81(cleaned value: a_alexandrov81))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov81(cleaned value: a_alexandrov81)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,419 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov82(cleaned value: a_alexandrov82))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov82(cleaned value: a_alexandrov82)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,468 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov83(cleaned value: a_alexandrov83))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov83(cleaned value: a_alexandrov83)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,515 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov84(cleaned value: a_alexandrov84))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov84(cleaned value: a_alexandrov84)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,563 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov85(cleaned value: a_alexandrov85))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov85(cleaned value: a_alexandrov85)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,611 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov86(cleaned value: a_alexandrov86))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov86(cleaned value: a_alexandrov86)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,658 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov87(cleaned value: a_alexandrov87))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov87(cleaned value: a_alexandrov87)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,706 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov88(cleaned value: a_alexandrov88))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov88(cleaned value: a_alexandrov88)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,753 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov89(cleaned value: a_alexandrov89))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov89(cleaned value: a_alexandrov89)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,803 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov90(cleaned value: a_alexandrov90))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov90(cleaned value: a_alexandrov90)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,850 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov91(cleaned value: a_alexandrov91))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov91(cleaned value: a_alexandrov91)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,882 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov92(cleaned value: a_alexandrov92))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov92(cleaned value: a_alexandrov92)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,929 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov93(cleaned value: a_alexandrov93))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov93(cleaned value: a_alexandrov93)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:19,961 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov94(cleaned value: a_alexandrov94))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov94(cleaned value: a_alexandrov94)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,009 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov95(cleaned value: a_alexandrov95))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov95(cleaned value: a_alexandrov95)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,057 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov96(cleaned value: a_alexandrov96))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov96(cleaned value: a_alexandrov96)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,105 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov97(cleaned value: a_alexandrov97))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov97(cleaned value: a_alexandrov97)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,137 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov98(cleaned value: a_alexandrov98))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov98(cleaned value: a_alexandrov98)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,184 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov99(cleaned value: a_alexandrov99))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov99(cleaned value: a_alexandrov99)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,234 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov100(cleaned value: a_alexandrov100))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov100(cleaned value: a_alexandrov100)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,281 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov101(cleaned value: a_alexandrov101))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov101(cleaned value: a_alexandrov101)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,329 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov102(cleaned value: a_alexandrov102))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov102(cleaned value: a_alexandrov102)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,377 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov103(cleaned value: a_alexandrov103))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov103(cleaned value: a_alexandrov103)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,425 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov104(cleaned value: a_alexandrov104))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov104(cleaned value: a_alexandrov104)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,472 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov105(cleaned value: a_alexandrov105))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov105(cleaned value: a_alexandrov105)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,505 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov106(cleaned value: a_alexandrov106))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov106(cleaned value: a_alexandrov106)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,553 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov107(cleaned value: a_alexandrov107))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov107(cleaned value: a_alexandrov107)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,601 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov108(cleaned value: a_alexandrov108))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov108(cleaned value: a_alexandrov108)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,632 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov109(cleaned value: a_alexandrov109))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov109(cleaned value: a_alexandrov109)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,680 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov110(cleaned value: a_alexandrov110))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov110(cleaned value: a_alexandrov110)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,727 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov111(cleaned value: a_alexandrov111))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov111(cleaned value: a_alexandrov111)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,776 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov112(cleaned value: a_alexandrov112))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov112(cleaned value: a_alexandrov112)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,808 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov113(cleaned value: a_alexandrov113))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov113(cleaned value: a_alexandrov113)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,856 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov114(cleaned value: a_alexandrov114))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov114(cleaned value: a_alexandrov114)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,903 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov115(cleaned value: a_alexandrov115))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov115(cleaned value: a_alexandrov115)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,951 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov116(cleaned value: a_alexandrov116))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov116(cleaned value: a_alexandrov116)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:20,999 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov117(cleaned value: a_alexandrov117))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov117(cleaned value: a_alexandrov117)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,034 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov118(cleaned value: a_alexandrov118))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov118(cleaned value: a_alexandrov118)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,078 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov119(cleaned value: a_alexandrov119))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov119(cleaned value: a_alexandrov119)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,126 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov120(cleaned value: a_alexandrov120))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov120(cleaned value: a_alexandrov120)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,174 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov121(cleaned value: a_alexandrov121))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov121(cleaned value: a_alexandrov121)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,206 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov122(cleaned value: a_alexandrov122))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov122(cleaned value: a_alexandrov122)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,253 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov123(cleaned value: a_alexandrov123))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov123(cleaned value: a_alexandrov123)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,300 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov124(cleaned value: a_alexandrov124))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov124(cleaned value: a_alexandrov124)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,349 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov125(cleaned value: a_alexandrov125))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov125(cleaned value: a_alexandrov125)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,396 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov126(cleaned value: a_alexandrov126))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov126(cleaned value: a_alexandrov126)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,444 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov127(cleaned value: a_alexandrov127))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov127(cleaned value: a_alexandrov127)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,492 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov128(cleaned value: a_alexandrov128))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov128(cleaned value: a_alexandrov128)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,540 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov129(cleaned value: a_alexandrov129))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov129(cleaned value: a_alexandrov129)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,588 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov130(cleaned value: a_alexandrov130))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov130(cleaned value: a_alexandrov130)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,635 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov131(cleaned value: a_alexandrov131))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov131(cleaned value: a_alexandrov131)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,683 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov132(cleaned value: a_alexandrov132))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov132(cleaned value: a_alexandrov132)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,731 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov133(cleaned value: a_alexandrov133))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov133(cleaned value: a_alexandrov133)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,778 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov134(cleaned value: a_alexandrov134))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov134(cleaned value: a_alexandrov134)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,826 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov135(cleaned value: a_alexandrov135))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov135(cleaned value: a_alexandrov135)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,873 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov136(cleaned value: a_alexandrov136))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov136(cleaned value: a_alexandrov136)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,922 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov137(cleaned value: a_alexandrov137))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov137(cleaned value: a_alexandrov137)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:21,969 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov138(cleaned value: a_alexandrov138))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov138(cleaned value: a_alexandrov138)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:22,016 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov139(cleaned value: a_alexandrov139))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov139(cleaned value: a_alexandrov139)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:22,064 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov140(cleaned value: a_alexandrov140))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov140(cleaned value: a_alexandrov140)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:22,111 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov141(cleaned value: a_alexandrov141))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov141(cleaned value: a_alexandrov141)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:22,143 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov142(cleaned value: a_alexandrov142))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov142(cleaned value: a_alexandrov142)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:22,191 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov143(cleaned value: a_alexandrov143))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov143(cleaned value: a_alexandrov143)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:22,242 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov144(cleaned value: a_alexandrov144))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov144(cleaned value: a_alexandrov144)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:22,288 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov145(cleaned value: a_alexandrov145))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov145(cleaned value: a_alexandrov145)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:22,337 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov146(cleaned value: a_alexandrov146))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov146(cleaned value: a_alexandrov146)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:22,383 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov147(cleaned value: a_alexandrov147))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov147(cleaned value: a_alexandrov147)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:22,431 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov148(cleaned value: a_alexandrov148))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov148(cleaned value: a_alexandrov148)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:07:22,479 - ERROR - Error while calling Moodle API: ('Error calling Moodle API\n', {'exception': 'invalid_parameter_exception', 'errorcode': 'invalidparameter', 'message': "Invalid parameter value detected (The field 'username' value is invalid: A_alexandrov149(cleaned value: a_alexandrov149))", 'debuginfo': "The field 'username' value is invalid: A_alexandrov149(cleaned value: a_alexandrov149)"}) - C:\Users\epira\Downloads\MoodleAPI-master\main.py:326
2023-10-01 12:08:50,862 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:508
2023-10-01 12:08:51,061 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 12:08:54,443 - INFO - New user registered: Alexandrov Andrey with login a_alexandrov and enrolled to the course Test123122 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 12:08:55,772 - INFO - New user registered: Alexandrov Andrey with login a_alexandrov1 and enrolled to the course Test123123 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 12:08:56,964 - INFO - New user registered: Alexandrov Andrey with login a_alexandrov2 and enrolled to the course Test123124 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 12:08:58,234 - INFO - New user registered: Alexandrov Andrey with login a_alexandrov3 and enrolled to the course Test123125 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 16:57:38,674 - INFO - New user registered: Alexandrov Andrey with login a_alexandrov4 and enrolled to the course Test123122 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 16:57:40,636 - INFO - New user registered: Alexandrov Andrey with login a_alexandrov5 and enrolled to the course Test123123 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 16:57:42,107 - INFO - New user registered: Alexandrov Andrey with login a_alexandrov6 and enrolled to the course Test123124 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 16:57:43,650 - INFO - New user registered: Alexandrov Andrey with login a_alexandrov7 and enrolled to the course Test123125 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 16:57:45,084 - INFO - New user registered: Alexandrov Andrey with login a_alexandrov8 and enrolled to the course Test123126 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:452
2023-10-01 17:04:05,281 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:04:08,275 - ERROR - Error while getting data from the sheet Осокин: name 'col_name_fio' is not defined - C:\Users\epira\Downloads\MoodleAPI-master\main.py:266
2023-10-01 17:05:17,067 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:300
2023-10-01 17:05:43,612 - ERROR - Error while enrolling user a_alexandrov4 to the course: name 'roleid' is not defined - C:\Users\epira\Downloads\MoodleAPI-master\main.py:243
2023-10-01 17:05:43,612 - INFO - New user registered: Alexandrov Andrey with login a_alexandrov4 and enrolled to the course Test123122 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:244
2023-10-01 17:05:43,613 - ERROR - Error while sending mail: name 'mail_subject' is not defined. Trying again - C:\Users\epira\Downloads\MoodleAPI-master\email_utils.py:83
2023-10-01 17:05:43,613 - INFO - Restarting program... - C:\Users\epira\Downloads\MoodleAPI-master\email_utils.py:84
2023-10-01 17:05:43,613 - ERROR - Error while sending email to user: name 'sleep' is not defined - C:\Users\epira\Downloads\MoodleAPI-master\main.py:250
2023-10-01 17:08:49,030 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:08:49,033 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:300
2023-10-01 17:08:52,368 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 'module' object is not subscriptable - C:\Users\epira\Downloads\MoodleAPI-master\main.py:295
2023-10-01 17:09:36,026 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 'module' object is not subscriptable - C:\Users\epira\Downloads\MoodleAPI-master\main.py:295
2023-10-01 17:10:47,487 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:10:47,489 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:303
2023-10-01 17:10:50,667 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 'module' object is not subscriptable - C:\Users\epira\Downloads\MoodleAPI-master\main.py:298
2023-10-01 17:11:52,860 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:11:52,862 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:303
2023-10-01 17:11:56,574 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 'module' object is not subscriptable - C:\Users\epira\Downloads\MoodleAPI-master\main.py:298
2023-10-01 17:12:26,500 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:12:26,503 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:300
2023-10-01 17:12:29,742 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): 'module' object is not subscriptable - C:\Users\epira\Downloads\MoodleAPI-master\main.py:295
2023-10-01 17:13:25,977 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:13:25,980 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:13:29,810 - ERROR - Error while creating user with email 101@gmail.com: 'auth' - C:\Users\epira\Downloads\MoodleAPI-master\main.py:207
2023-10-01 17:13:30,045 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): {'code': 400, 'message': 'Invalid values[3][0]: list_value {\n  values {\n    string_value: "error"\n  }\n  values {\n    string_value: ""\n  }\n  values {\n    string_value: "ERROR - there is problem while user registration"\n  }\n}\n', 'status': 'INVALID_ARGUMENT'} - C:\Users\epira\Downloads\MoodleAPI-master\main.py:294
2023-10-01 17:17:03,698 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:17:03,703 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:17:07,450 - ERROR - Error while creating user with email 101@gmail.com: 'auth' - C:\Users\epira\Downloads\MoodleAPI-master\main.py:207
2023-10-01 17:17:07,686 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): {'code': 400, 'message': 'Invalid values[3][0]: list_value {\n  values {\n    string_value: "error"\n  }\n  values {\n    string_value: ""\n  }\n  values {\n    string_value: "ERROR - there is problem while user registration"\n  }\n}\n', 'status': 'INVALID_ARGUMENT'} - C:\Users\epira\Downloads\MoodleAPI-master\main.py:294
2023-10-01 17:18:15,465 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:18:15,469 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:18:19,747 - ERROR - Error while creating user with email 101@gmail.com: 'auth' - C:\Users\epira\Downloads\MoodleAPI-master\main.py:207
2023-10-01 17:18:19,975 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): {'code': 400, 'message': 'Invalid values[3][0]: list_value {\n  values {\n    string_value: "error"\n  }\n  values {\n    string_value: ""\n  }\n  values {\n    string_value: "ERROR - there is problem while user registration"\n  }\n}\n', 'status': 'INVALID_ARGUMENT'} - C:\Users\epira\Downloads\MoodleAPI-master\main.py:294
2023-10-01 17:18:34,673 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:18:44,938 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:18:44,942 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:18:48,620 - ERROR - Error while creating user with email 101@gmail.com: 'auth' - C:\Users\epira\Downloads\MoodleAPI-master\main.py:207
2023-10-01 17:18:48,851 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): {'code': 400, 'message': 'Invalid values[3][0]: list_value {\n  values {\n    string_value: "error"\n  }\n  values {\n    string_value: ""\n  }\n  values {\n    string_value: "ERROR - there is problem while user registration"\n  }\n}\n', 'status': 'INVALID_ARGUMENT'} - C:\Users\epira\Downloads\MoodleAPI-master\main.py:294
2023-10-01 17:19:08,034 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:19:08,039 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:19:11,905 - ERROR - Error while creating user with email 101@gmail.com: 'auth' - C:\Users\epira\Downloads\MoodleAPI-master\main.py:207
2023-10-01 17:19:12,118 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): {'code': 400, 'message': 'Invalid values[3][0]: list_value {\n  values {\n    string_value: "error"\n  }\n  values {\n    string_value: ""\n  }\n  values {\n    string_value: "ERROR - there is problem while user registration"\n  }\n}\n', 'status': 'INVALID_ARGUMENT'} - C:\Users\epira\Downloads\MoodleAPI-master\main.py:294
2023-10-01 17:19:29,322 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:19:29,326 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:19:33,186 - ERROR - Error while creating user with email 101@gmail.com: 'auth' - C:\Users\epira\Downloads\MoodleAPI-master\main.py:207
2023-10-01 17:19:33,410 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): {'code': 400, 'message': 'Invalid values[3][0]: list_value {\n  values {\n    string_value: "error"\n  }\n  values {\n    string_value: ""\n  }\n  values {\n    string_value: "ERROR - there is problem while user registration"\n  }\n}\n', 'status': 'INVALID_ARGUMENT'} - C:\Users\epira\Downloads\MoodleAPI-master\main.py:294
2023-10-01 17:24:48,819 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:24:48,824 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:24:58,083 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:24:58,087 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:25:07,669 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:25:07,674 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:25:13,019 - ERROR - Error while enrolling user a_alexandrov9 to the course: name 'roleid' is not defined - C:\Users\epira\Downloads\MoodleAPI-master\main.py:242
2023-10-01 17:25:13,019 - INFO - New user registered: Alexandrov Andrey with login a_alexandrov9 and enrolled to the course Test123122 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:243
2023-10-01 17:25:13,020 - ERROR - Error while sending mail: 'ConfigParser' object has no attribute 'mail_subject'. Trying again - C:\Users\epira\Downloads\MoodleAPI-master\email_utils.py:87
2023-10-01 17:25:13,020 - INFO - Restarting program... - C:\Users\epira\Downloads\MoodleAPI-master\email_utils.py:88
2023-10-01 17:27:22,884 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:27:22,887 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:27:26,201 - INFO - New user registered: Alexandrov Andrey with login a_alexandrov9 and enrolled to the course Test123122 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:243
2023-10-01 17:28:13,602 - ERROR - Error while creating course Test12312312312: name 'category_id' is not defined - C:\Users\epira\Downloads\MoodleAPI-master\main.py:233
2023-10-01 17:28:13,830 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): {'code': 400, 'message': 'Invalid values[4][0]: list_value {\n  values {\n    string_value: "error"\n  }\n  values {\n    string_value: ""\n  }\n  values {\n    string_value: "ERROR - there is problem while creating course"\n  }\n}\n', 'status': 'INVALID_ARGUMENT'} - C:\Users\epira\Downloads\MoodleAPI-master\main.py:294
2023-10-01 17:28:57,960 - ERROR - Error while creating course Test12309: name 'category_id' is not defined - C:\Users\epira\Downloads\MoodleAPI-master\main.py:233
2023-10-01 17:28:58,184 - ERROR - Error while working with Google Sheets (list <Worksheet 'Осокин' id:1518056643>): {'code': 400, 'message': 'Invalid values[4][0]: list_value {\n  values {\n    string_value: "error"\n  }\n  values {\n    string_value: ""\n  }\n  values {\n    string_value: "ERROR - there is problem while creating course"\n  }\n}\n', 'status': 'INVALID_ARGUMENT'} - C:\Users\epira\Downloads\MoodleAPI-master\main.py:294
2023-10-01 17:29:06,964 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:29:06,967 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:29:10,189 - INFO - New course created: Test12309 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:226
2023-10-01 17:29:10,978 - INFO - New user registered: Stepanov Dmitriy with login d_stepanov1 and enrolled to the course Test12309 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:243
2023-10-01 17:29:35,524 - INFO - New user registered: Alexandrov Andrey with login borolev and enrolled to the course Test123124 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:243
2023-10-01 17:36:15,623 - INFO - file_cache is only supported with oauth2client<4.0.0 - C:\Users\epira\Downloads\MoodleAPI-master\moodle\Lib\site-packages\googleapiclient\discovery_cache\__init__.py:49
2023-10-01 17:36:15,625 - INFO - Program started successfully! - C:\Users\epira\Downloads\MoodleAPI-master\main.py:299
2023-10-01 17:36:19,314 - INFO - New user registered: Alexandrov Andrey with login borolev and enrolled to the course Test123124 - C:\Users\epira\Downloads\MoodleAPI-master\main.py:243
